### ISSUE TITLE
Cannot find name 'UIImage'

### ISSUE THREAD

**Issue by ezpzlmnsqzy1** [Akylas/nativescript-canvas#6]
### Please, provide the following version numbers that your issue occurs with:
- CLI: 5.4.2
- Cross-platform modules: 5.4.3
- Runtime(s): Android 5.4.0, ios 5.4.2
- Plugin(s): 
```
"dependencies": {
    "@angular/animations": "~8.0.0",
    "@angular/common": "~8.0.0",
    "@angular/compiler": "~8.0.0",
    "@angular/core": "~8.0.0",
    "@angular/forms": "~8.0.0",
    "@angular/http": "~8.0.0-beta.10",
    "@angular/platform-browser": "~8.0.0",
    "@angular/platform-browser-dynamic": "~8.0.0",
    "@angular/router": "~8.0.0",
    "nativescript-angular": "~8.0.0",
    "nativescript-camera": "^4.5.0",
    "nativescript-canvas": "^2.1.7",
    "nativescript-imagepicker": "^6.2.0",
    "nativescript-plugin-firebase": "^9.0.1",
    "nativescript-theme-core": "~1.0.4",
    "reflect-metadata": "~0.1.12",
    "rxjs": "~6.5.0",
    "tns-core-modules": "~5.4.0",
    "tns-platform-declarations": "^5.4.3",
    "zone.js": "~0.9.1"
  },
  "devDependencies": {
    "@angular/compiler-cli": "~8.0.0",
    "@ngtools/webpack": "~8.0.0",
    "nativescript-dev-typescript": "~0.10.0",
    "nativescript-dev-webpack": "~0.24.0"
  }
```
Edit: I also noticed there is no 'android' folder under 'node_modules\nativescript-camera\platforms' but there is an ios folder.

### Please, tell us how to recreate the issue in as much detail as possible. 
Ran `tns plugin add nativescript-canvas` in my project, followed by `tns run android`. During the run process the following is outputted: 
```
ERROR in node_modules/nativescript-canvas/canvas.d.ts(64,71): error TS2304: Cannot find name 'UIImage'.
node_modules/nativescript-canvas/canvas.d.ts(68,43): error TS2304: Cannot find name 'UIImage'.
node_modules/nativescript-canvas/canvas.d.ts(95,50): error TS2304: Cannot find name 'UIImage'.
node_modules/nativescript-canvas/canvas.d.ts(96,50): error TS2304: Cannot find name 'UIImage'.
```
The app still deploys to my phone and (generally) works, but none of the `Canvas` methods work.

**Comment by farfromrefug** [comment]
I can fix it in the future . Right now you can fix it by using ```tns-platform-declarations```

**Comment by farfromrefug** [comment]
Actually fixed in 2.1.8
